### PR TITLE
[AIRFLOW-3606] Fix Flake8 test & fix the Flake8 errors introduced since Flake8 test was broken

### PR DESCRIPTION
**Pull request by XD-DENG**
### Jira

  - https://issues.apache.org/jira/browse/AIRFLOW-3606

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

1. **The `flake8 test` in the Travis CI was broken since https://github.com/apache/incubator-airflow/pull/4361 (https://github.com/apache/incubator-airflow/commit/7a6acbf5b343e4a6895d1cc8af75ecc02b4fd0e8 )**

2. Some Flake8 errors (code style/quality issues. found in 10 files) were introduce since flake8 test was broken.

In this PR, I reverted https://github.com/apache/incubator-airflow/pull/4361 and fixed all the Flake8 errors introduce into master branch since the `flake8 test` was broken (in the past 9 days).

### CODE REVIEW

**Comment by XD-DENG** [comment]
Hi @ashb , @Fokko , @kaxil , @feng-tao , may any of you have a quick check on this?

**The flake8 test in the Travis CI was broken since https://github.com/apache/incubator-airflow/pull/4361** .

You can refer to this sample commit https://github.com/XD-DENG/incubator-airflow/commit/ba046e3901ae in my another test branch, in which I purposely made some Flake8 errors, while this test still passed with green light (https://travis-ci.org/XD-DENG/incubator-airflow/jobs/474002074). https://github.com/apache/incubator-airflow/pull/4361 actually "turned off" the Flake8 test.

It would be good if we can fix this issue quickly, to avoid any potential Flake8 errors in new PRs to be introduced into master branch.

CC @holdenk who authored https://github.com/apache/incubator-airflow/pull/4361. Please let me know if I misunderstood or missed anything.

Thanks!

File: .flake8 [hunk 1]
```diff
@@ -1,4 +1,3 @@
 [flake8]
 max-line-length = 110
 ignore = E731
-exclude = .git, env, venv, .venv, VENV, .tox, .eggs, .*
```
Comment (holdenk) [inline .flake8:4]: Is the problem from the ".*" or does any exclude cause this problem?

Comment (XD-DENG) [reply .flake8:4]: I'm not completely sure, but I believe it's the `.*`.

But before we trouble-shoot thoroughly, I would suggest to revert the whole line to ensure that Travis CI works as expected.

File: airflow/bin/cli.py [hunk 2]
```diff
@@ -69,7 +69,6 @@
 from airflow.www_rbac.app import create_app as create_app_rbac
 from airflow.www_rbac.app import cached_appbuilder
 
-from sqlalchemy import func
```
Comment (feng-tao) [inline airflow/bin/cli.py:72]: the commit of this line is 1 year ago(https://github.com/apache/incubator-airflow/commit/b9a87a07e3b6dbd965c91f09d0f1c10281f628f9).

Comment (feng-tao) [reply airflow/bin/cli.py:72]: are we sure this line is not using?

Comment (feng-tao) [reply airflow/bin/cli.py:72]: is it because we remove the func usage in some commits recently?

Comment (XD-DENG) [reply airflow/bin/cli.py:72]: Exactly. The func usage was removed in https://github.com/apache/incubator-airflow/commit/53e707428826d603f43aebb73e18fe077881c4b3#diff-1c2404a3a60f829127232842250ff406 which was merged three days ago.

**Review by feng-tao — COMMENTED** [review]
overall lgtm, but I haven't taken a look in previous pr on why it breaks flake8 test. Have a small qq.

**Comment by feng-tao** [comment]
LGTM. thanks @XD-DENG

File: .flake8 (hunk 1 above)
Comment (holdenk) [reply .flake8:4]: Makes sense :)

**Pull request merged**